experimenter/experimenter/nimbus_ui_new/filtersets.py

```diff
@@ -0,0 +1,281 @@
+import django_filters
+from django import forms
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models import Q
+from django.db.models.functions import Concat
+
+from experimenter.base.models import Country, Language, Locale
+from experimenter.experiments.models import NimbusExperiment, NimbusFeatureConfig
+from experimenter.projects.models import Project
+from experimenter.targeting.constants import TargetingConstants
+
+
+class StatusChoices(models.TextChoices):
+    DRAFT = NimbusExperiment.Status.DRAFT
+    PREVIEW = NimbusExperiment.Status.PREVIEW
+    LIVE = NimbusExperiment.Status.LIVE
+    COMPLETE = NimbusExperiment.Status.COMPLETE
+    REVIEW = "Review"
+    ARCHIVED = "Archived"
```

Comment:
we don't have constants for `Review` and `Archived` like we have for Draft etc `DRAFT = NimbusExperiment.Status.DRAFT`?

Reply:
No I don't think so, but it's okay these are just labels for the filter, they can be anything.